src/main/java/calculator/controller/calculatorSystem/CalculatorSystem.java

```diff
@@ -0,0 +1,14 @@
+package calculator.controller.calculatorSystem;
+
+import calculator.model.inputValue.InputValue;
+import calculator.view.screen.Screen;
+
+public class CalculatorSystem {
+    private InputValue inputValue;
+    private Screen screen = new Screen();

```

Comment:
객체가 중간에 바뀔 일이 없다면 `final` 키워드를 붙이는건 어떨까요?

```suggestion
    private final InputValue inputValue;
    private final Screen screen = new Screen();
```

Reply:
그래도 될 것 같아요!

Reply:
제가 피드백 받았던 내용인데, 민준님께서도 한 번 생각해보시면 좋을 것 같아 남깁니다!

`Controller가 '다른 객체를 생성하는 책임'을 가져야 할까?`